tb/cocotb/soc/test_axil_to_apb.py

```
@@ -0,0 +1,292 @@
+Tests:
+    T1  test_write_roundtrip            — AXI write → APB SETUP/ACCESS → bresp OKAY
+    T2  test_read_roundtrip             — AXI read → APB SETUP/ACCESS → rdata/rresp OKAY
+    T3  test_pslverr_propagation        — force_slverr=1 → bresp/rresp = SLVERR (2'b10)
+    T4  test_aw_before_w               — AW arrives one cycle before W; bridge buffers AW
+    T5  test_w_before_aw               — W arrives one cycle before AW; bridge buffers W
+    T6  test_back_to_back_write_read   — back-to-back write then read; read waits for write
+"""
+
+import cocotb
+from cocotb.clock import Clock
+from cocotb.triggers import RisingEdge, Combine
```

Flake8 (7.3.0)

[error] 'cocotb.triggers.Combine' imported but unused

(F401)